src/trackers/classes/commonRequest.js

```diff
@@ -66,6 +67,11 @@ function _update (commonReq, newReq, site) {
                 commonReq.cnames.push(record)
             }
         }
+
+
+        if (newReq.responseHash && !commonReq.responseHashes.includes(newReq.responseHash)) {
+            commonReq.responseHashes.push(newReq.responseHash)
```

Comment:
nit: it might be a bit more efficient to use Set for `commonReq.responseHashes` and then

```suggestion
        if (newReq.responseHash) {
            commonReq.responseHashes.add(newReq.responseHash)
```

Reply:
Makes write the data to a file more difficult though.

Reply:
Yeah, you have to remember to `Array.from()` before writing to file. It's stupid that this doesn't searialize to an array by default.